src/organizations/operations/update_organization.rs

```diff
@@ -0,0 +1,147 @@
+use std::collections::HashSet;
+
+use async_trait::async_trait;
+use reqwest::StatusCode;
+use serde::Serialize;
+use thiserror::Error;
+
+use crate::organizations::{Organization, OrganizationId, Organizations};
+use crate::{WorkOsError, WorkOsResult};
+
+/// The options for [`UpdateOrganization`].
+#[derive(Debug, Serialize)]
+pub struct UpdateOrganizationOptions<'a> {
+    /// The ID of the organization passed in the URL.
+    #[serde(skip_serializing)]
+    pub organization_id: &'a OrganizationId,
+
+    /// The name of the organization.
+    pub name: &'a str,
```

Comment:
Let's make the name optional so that the developer doesn't need to provide it if they don't want to:

```suggestion
    pub name: Option<&'a str>,
```